packages/contracts/npm/artifacts.ts

```diff
@@ -15,20 +15,40 @@ import * as DAORegistry from '../artifacts/src/framework/dao/DAORegistry.sol/DAO
 import * as TokenFactory from '../artifacts/src/framework/utils/TokenFactory.sol/TokenFactory.json';
```

Comment:
Do we need the TokenFactory artifact?

Reply:
since this is a contract and we still have it,
even it is not deployed, i would say let's publish it just in case